server.js

```diff
@@ -249,9 +249,22 @@ app
           .getJobs()
           .then((jobs) => {
             let filteredJobs = jobs.map((job) => {
+              let date = new Date(job.timestamp);
               return {
                 id: job.id,
                 title: _.get(job.data.details, bookTitle[req.query.queue_name]),
+                userName: job.data.userName ? job.data.userName : "-",
+                timestamp:
+                  date.getUTCFullYear() +
+                  "-" +
+                  date.getUTCMonth() +
+                  "-" +
+                  date.getUTCDate() +
+                  " " +
+                  date.getUTCHours() +
+                  ":" +
+                  date.getUTCMinutes() +
```

Comment:
`Same as above. We should handle single digit for `getUTCMinutes()`. `getUTCHours()` could be single and the time can be represented as `5:04 (UTC)`